test/groovy/com/sap/piper/MapUtilsTest.groovy

```diff
@@ -51,6 +54,15 @@ class MapUtilsTest {
         assert m == [a: 'replaced', m: [b: 'replaced', c: 'otherString']]
     }
 
+    @Test
+    void testGetByPath() {
+        Map m = [trees: [oak: 5, beech :1], flowers:[rose: 23]]
+
+        assertThat(MapUtils.getByPath(m, 'flowers'), is([rose: 23]))
+        assertThat(MapUtils.getByPath(m, 'trees/oak'), is(5))
+        assertThat(MapUtils.getByPath(m, 'trees/palm'), is(null))
```

Comment:
[nit] we cannot distiguish between a path which has an associated `null` value and a non resolvable path ... <sigh ...>

Reply:
So what would you expect, an NPE?

Reply:
At least for the `config` use case we applied `pruneNulls` so we can expect no `null` values.

Reply:
> At least

yes, lets keep it as it is. The perfect - but also awkward - way would be to provide also a possibility to check if a given path exists (only the path, regardless of the vallue)